fastestimator/estimator.py

```diff
@@ -194,23 +206,21 @@ class System:
     def __init__(self,
```

Comment:
System class should move to its own file, possibly inside Util folder since we don't want pipeline and/or network to import from estimator

Reply:
I think we won't import system anywhere else.  if that's the case is it ok to put it inside estimator?

Reply:
i think it will probably play a role in the Summary object when we bring that functionality back

Reply:
even if we don't though i think it makes more sense to move it from an organization perspective

Reply:
sure